### ISSUE TITLE
Module not found on Android only

### ISSUE THREAD

**Issue by dapriett** [NativeScript/NativeScript#822]
Hello,

For some reason Android doesn't like the npm module [key-tree-store](https://www.npmjs.com/package/key-tree-store), but works fine on iOS.  I was using another module which had this as a dependency.  I tried installing it directly and requiring it, I get the same error.  It looks like it's there, so not sure why it can't find it - has no issues finding only modules I've added.  Below is the error:

```
09-24 11:04:59.874: E/TNS.Native(4336): Error: Module "key-tree-store" not found
09-24 11:04:59.874: E/TNS.Native(4336): File: "/data/data/com.test.app/files/app/models/base-model.js, line: 0, column: 282
09-24 11:04:59.874: E/TNS.Native(4336): StackTrace: 
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'require', file:'/data/data/com.test.app/files/app/models/base-model.js', line: 2, column: 283
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'', file:'/data/data/com.test.app/files/app/models/base-model.js', line: 19, column: 11
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'require', file:'/data/data/com.test.app/files/app/models/user.js', line: 2, column: 277
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'', file:'/data/data/com.test.app/files/app/models/user.js', line: 1, column: 17
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'require', file:'/data/data/com.test.app/files/app/components/sidedrawer.js', line: 2, column: 291
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'', file:'/data/data/com.test.app/files/app/components/sidedrawer.js', line: 2, column: 17
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'require', file:'/data/data/com.test.app/files/app/tns_modules/ui/builder/builder.js', line: 2, column: 312
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'loadCustomComponent', file:'/data/data/com.test.app/files/app/tns_modules/ui/builder/builder.js', line: 152, column: 26
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'', file:'/data/data/com.test.app/files/app/tns_modules/ui/builder/builder.js', line: 88, column: 39
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'', file:'/data/data/com.test.app/files/app/tns_modules/xml/xml.js', line: 106, column: 13
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'EasySAXParser.parse', file:'/data/data/com.test.app/files/app/tns_modules/js-libs/easysax/easysax.js', line: 720, column: 23
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'XmlParser.parse', file:'/data/data/com.test.app/files/app/tns_modules/xml/xml.js', line: 153, column: 22
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'parseInternal', file:'/data/data/com.test.app/files/app/tns_modules/ui/builder/builder.js', line: 136, column: 19
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'loadInternal', file:'/data/data/com.test.app/files/app/tns_modules/ui/builder/builder.js', line: 207, column: 27
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'load', file:'/data/data/com.test.app/files/app/tns_modules/ui/builder/builder.js', line: 191, column: 27
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'pageFromBuilder', file:'/data/data/com.test.app/files/app/tns_modules/ui/frame/frame-common.js', line: 71, column: 27
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'resolvePageFromEntry', file:'/data/data/com.test.app/files/app/tns_modules/ui/frame/frame-common.js', line: 52, column: 20
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'Frame.navigate', file:'/data/data/com.test.app/files/app/tns_modules/ui/frame/frame-common.js', line: 110, column: 20
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'AndroidApplication.getActivity', file:'/data/data/com.test.app/files/app/tns_modules/application/application.js', line: 140, column: 26
09-24 11:04:59.874: E/TNS.Native(4336): 	Frame: function:'app.init.getActivity', file:'/data/data/com.test.app/files/app/tns_modules/application/application.js', line: 111, column: 32
09-24 11:04:59.908: E/EGL_emulation(755): tid 755: eglCreateSyncKHR(1209): error 0x3004 (EGL_BAD_ATTRIBUTE)
```

**Comment by atanasovg** [comment]
Hey @dapriett, thanks for reporting this, the issue is already fixed in the [Android Runtime](https://github.com/NativeScript/android-runtime/issues/190) and it will be available with the 1.4 release (expected within two-3 weeks). As a temporary workaround you may add `.js` to the main entry in the `package.json` of the module.

Closing.

**Comment by dapriett** [comment]
Thanks @atanasovg!
